src/lib/Behat/PageElement/Notification.php

```diff
@@ -0,0 +1,28 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: maciejtyrala
+ * Date: 26/01/2018
+ * Time: 11:36
+ */
+
+namespace EzSystems\EzPlatformAdminUi\Behat\PageElement;
+
+
+class Notification extends Element
+{
+    protected $fields = ['successAlert' => '.alert.alert-success.alert-dismissible.fade.show',
+                    ];
+    public const ELEMENT_NAME = 'Notification';
+
+    public function verifyVisibility(): void
+    {
+        $this->context->waitUntilElementIsVisible($this->fields['successAlert'], $this->defaultTimeout);
```

Comment:
You're right, I'll fix it. I was assume that there will be potentially more success alerts than the failure ones.
Yet, it's not something critical, because this element isn't assigned to any page, just like upperMenu, so verifyVisibility method will be call only on demand.